packages/device_info_plus_platform_interface/lib/model/web_browser_info.dart

```diff
@@ -139,8 +139,10 @@ class WebBrowserInfo {
     } else if (userAgent.contains('Trident')) {
       return BrowserName.msie;
       // "Mozilla/5.0 (Windows NT 10.0; WOW64; Trident/7.0; .NET4.0C; .NET4.0E; Zoom 3.6.0; wbx 1.0.0; rv:11.0) like Gecko"
-    } else if (userAgent.contains('Edge')) {
+    } else if (userAgent.contains('Edge') || userAgent.contains('Edg')) {
```

Comment:
is userAgent the whole string? is checking for 'Edg' alone enough?

Reply:
This is token that vendors add. it must be adequate.  according to this https://docs.microsoft.com/en-us/microsoft-edge/web-platform/user-agent-string

Reply:
`userAgent.contains('Edg')` will return true if `'Edge'` is a part of the string as well, no?
`userAgent.contains('Edge')` seems redundant.

Reply:
yes, that's correct, Good catch. I remove it now